modules/meeting/app/forms/meeting/time_group.rb

```diff
@@ -64,7 +64,7 @@ class Meeting::TimeGroup < ApplicationForm
         label: Meeting.human_attribute_name(:duration),
         visually_hide_label: false,
         required: true,
-        caption: I18n.t("text_in_hours"),
+        trailing_visual: { text: { text: I18n.t("text_hours") } },
```

Comment:
Hi @bsatarnejad! There is already a translation in core that we can use here, I think: `datetime.units.hour`:

```yaml
    units:
      hour:
        one: "hour"
        other: "hours"
```

So you should be able to call `I18n.t("datetime.units.hour", count: 2)` or `I18n.t("datetime.units.hour.other")` (I'm not sure which is the more correct use of the I18n API though)

Reply:
Hi @myabc 
Thanks for the hint 👍 
We may use units more in the future, so it would be better to define them globally.